src/js/components/DescriptionList.js

```diff
@@ -2,16 +2,16 @@ import React from 'react';
 
 class DescriptionList extends React.Component {
   getHeadline() {
-    let {headline} = this.props;
+    let {headline, headlineClassName} = this.props;
     if (!headline) {
       return null;
     }
 
     // Wrap in headline element and classes
     return (
-      <h6 className="flush-top">
+      <h5 className={headlineClassName}>
```

Comment:
Have you verified this change with design and checked the usage everywhere, that it still looks how we want it?